src/Microsoft.NetCore.Analyzers/Core/Security/DoNotUseInsecureDeserializerWithoutBinderBase.cs

```diff
@@ -97,7 +97,7 @@ public sealed override void Initialize(AnalysisContext context)
                                 {
                                     IInvocationOperation invocationOperation =
                                         (IInvocationOperation)operationAnalysisContext.Operation;
-                                    if (invocationOperation.TargetMethod.ContainingType == deserializerTypeSymbol
```

Comment:
Did we add any tests for this specific change?

Reply:
ReadObject_Stream_* tests.  The ReadObject(Stream) overload is inherited from XmlObjectSerializer.